infrastructure/crowdin/pushTranslations.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+#$3 github API Key
+pull_request() {
+  PR="{\"title\": \"[${BRANCH_NAME}] Translations update\",\"body\": \"Latest translations made in [Crowdin](https://crowdin.com/project/bonita-bpm)\n!skipTests\", \"head\": \"feat/${BRANCH_NAME}/update-translations\", \"base\": \"${BRANCH_NAME}\"}"
```

Comment:
⚠️ url should be https://crowdin.com/project/bonita (bonita-bpm is a former name and redirect to bonita)

you should also removed `!skipTests` which is not used by the project jenkinsfile to skip tests when running a PR